src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerManager.AnalyzerExecutionContext.cs

```diff
@@ -173,6 +173,16 @@ void processMembers(IEnumerable<ISymbol> members)
                             {
                                 memberSet = memberSet ?? new HashSet<ISymbol>();
                                 memberSet.Add(member);
+
+                                // Ensure that we include symbols for both parts of partial methods.
+                                if (member is IMethodSymbol method)
```

Comment:
are methods the only symbol that can be partial?

Reply:
I believe they are the only ones where they have a different symbol for other part of partial, unless I am unaware of another such symbol kind.